files/userdata_server

```diff
@@ -9,6 +9,10 @@ elif [ `command -v wget` ]; then
   wget -qO- https://releases.rancher.com/install-docker/${docker_version_server}.sh | sh
 fi
 
+### Customize Kernel parameters
+echo ${kernel_nf_conntrack_max} > /proc/sys/net/netfilter/nf_conntrack_max
```

Comment:
This should probably be a combination of putting the value in `/etc/sysctl.conf` and using `sysctl -p` to apply in case anyone decided to restart the droplet and expect it to still work. What do you think?

Reply:
Make sense